assets/dev/js/editor/components/browser-import/session.js

```diff
@@ -89,14 +89,18 @@ export default class Session {
 	 * @returns {*}
 	 */
 	resolve( containers ) {
+		if ( Object.values( containers ).some( ( element ) => 'section' === element.model.get( 'elType' ) ) ) {
```

Comment:
Can you scroll to the first new imported element (on after the import is completed)

Reply:
Done.